tests/e2e/specs/FirstStep.js

```diff
@@ -1,18 +1,19 @@
 module.exports = {
     '@tags': ['first'],
-    'FirstStep'(browser){
-        const yesRadioSelector = '#site-container > div.columns > div.stepper-container.column > div.step-container.current.enabled > div.step-content > div > div.field.not-selected > label > span.check'
-        const noRadioSelector = '#site-container > div.columns > div.stepper-container.column > div.step-container.current.enabled > div.step-content > div > div.field.not-selected > label > span.check'
-        const attributionHeader = '#site-container > div.columns > div.stepper-container.column > div:nth-child(2) > div > h5'
-        const ccLicenseHeader = '#site-container > div.columns > div.stepper-container.column > div:nth-child(2) > div'
+    'FirstStep'(browser) {
+        const radioSelector = 'div.field.not-selected > label > span.control-label > span'
+        const radioSelected = 'div.field.selected > label > span.control-label > span'
         const nextBtn = '.pagination-next'
-       
+        const stepTitle = 'div.step-header > h5'
+
         browser
             .init()
-            .click(yesRadioSelector)
-            .assert.visible(ccLicenseHeader, 'Click yes:Creative Commons License header is visible in the step below FirstStep')
-            .assert.visible(nextBtn, 'Next button is visible')
-            .click(noRadioSelector)
-            .assert.visible(attributionHeader, 'Click no:Attribution header is visible in the step below FirstStep')
-    }        
-}
\ No newline at end of file
+            .assert.containsText(stepTitle, 'Do you know which license you need?')
+            .assert.containsText(radioSelector, 'Yes. I know which license I need')
```

Comment:
You do not need to test the stepTitle/radioSelector tests as you are testing them in the unit tests. 
The texts that do need to be tested are:

- Heading of the step coming after the first step: it should be either 'Attribution' or 'Creative Commons License' based on the answer you give in the First Step (yes/no).
- Contents of `p.step-description` **after** the Next button has been clicked: it should be based on your answer (yes/no).

Reply:
Thanks for the clarification, I will modify my tests accordingly.